src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo__img {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__list .nav__item:not(:last-child) {
+  margin-right: 20px;
```

Comment:
Remember to be consistent with your vertical margins. It's a good practice to add margin only to the top or bottom (but not both) to avoid unexpected behavior due to margin collapsing. [Checklist item: Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both)]